internal/explorer/server.go

```diff
@@ -175,17 +176,31 @@ func (a *App) getNode(w http.ResponseWriter, r *http.Request) {
 func (a *App) getNodeStatus(w http.ResponseWriter, r *http.Request) {
 	enableCors(&w)
 
+	response := NodeStatus{Status: "loading"}
+
 	nodeID := mux.Vars(r)["node_id"]
-	_, err := a.getNodeData(nodeID, false)
+	value, err := a.GetRedisKey(fmt.Sprintf("GRID3NODE:%s", nodeID))
```

Comment:
I see this pattern a lot, wouldn't it be better if u have a method that `builds` the key for u so it's all in one place instead of copying the same string all over the place

```go
func getNodeKey(nodeID) string 
```